packages/edit-site/src/components/global-styles/screen-colors.js

```diff
@@ -29,6 +31,12 @@ function ScreenColors() {
 	const [ rawSettings ] = useGlobalSetting( '' );
 	const settings = useSettingsForBlockElement( rawSettings );
 
+	const colorVariations = useCurrentMergeThemeStyleVariationsWithUserConfig( {
+		property: 'color',
```

Comment:
I think I'll work on a variation of this function that accepts an array of properties. 

Here's twentytwentythree getting gradient bleed: 

https://github.com/WordPress/gutenberg/assets/6458278/dca25a59-3c5d-4340-9e68-42d977865718

Reply:
Actually, withdraw the above comment.

It's because I forgot to reinstate [removePropertyFromObject](https://github.com/WordPress/gutenberg/pull/56622/commits/1d25ce937aee5a9bb538b4d4f9c7dc94c7c745bd#diff-f27afb89c8e1505893fb5dd62b88e81570c0c4032195d2a5b79774a2b59de69bR61).

Sorry!

I'll reinstate.